docs/references/kontena-yml.md

```diff
@@ -74,7 +79,7 @@ instances: 1
 
 #### stateful
 
-Mark service as stateful (default: false). Kontena will create and automatically mount a data volume container for the service. This option also instructs the scheduler to bind the service instance to the scheduled host so that the volume can be mapped when the service is updated.
+Mark service as stateful (default: false). Kontena will create and automatically mount a data volume container for the service. This option also instructs the scheduler to not make any automatic scheduling decisions after initial deployment is done and bind the service instance to the scheduled host so that the volume can be mapped when the service is updated.
```

Comment:
This option binds each service instance to the host node that is first scheduled on to, so that the same volumes can be mapped whenever the service is updated. This option also instructs the scheduler to not make any automatic scheduling decisions for the service after initial deployment is done.